core/scripts.py

```diff
@@ -2905,45 +2899,113 @@ def inventory_report_query(self) -> str:
-                 left outer join (select ln.license_pool_id,
+             works w left outer join (select wg.work_id, string_agg(g.name, ',' order by g.name) as genres
+                                     from genres g,
+                                     workgenres wg
+                                     where g.id = wg.genre_id
+                                     group by wg.work_id) wg on w.id = wg.work_id,
+             editions e left outer join (select lp.presentation_edition_id,
                                          p.library_id,
                                          count(ln.id) active_loan_count
                                   from loans ln,
+                                       licensepools lp,
                                        patrons p,
                                        libraries l
                                   where p.id = ln.patron_id and
                                         p.library_id = l.id and
+                                        ln.license_pool_id = lp.id and
                                         l.id = :library_id
-                                  group by p.library_id, ln.license_pool_id) lib_loans on lp.id = lib_holds.license_pool_id
+                                  group by p.library_id, lp.presentation_edition_id) lib_loans
+                                  on e.id = lib_loans.presentation_edition_id,
+             identifiers i,
+             (select ilc.parent_id,
+                      count(ilc.parent_id) > 1 is_shared_collection
+              from integration_library_configurations ilc,
+                   integration_configurations i,
+                   collections c
+              where c.integration_configuration_id = i.id  and
+                    i.id = ilc.parent_id group by ilc.parent_id) collection_sharing,
+             licensepools lp left outer join (select license_pool_id,
+                                                     checkouts_left,
+                                                     expires,
+                                                     terms_concurrency
+                                              from licenses) l on lp.id = l.license_pool_id
+        where lp.identifier_id = i.id and
+              e.primary_identifier_id = i.id and
+              e.id = w.presentation_edition_id and
+              d.id = e.data_source_id and
+              c.id = lp.collection_id and
+              c.integration_configuration_id = ic.id and
+              ic.id = il.parent_id and
+              ic.id = collection_sharing.parent_id and
+              ic.goal = 'LICENSE_GOAL' and
+              (ic.settings -> 'include_in_inventory_report' = 'true' or
+              ic.settings -> 'include_in_inventory_report' is null) and
+              il.library_id = lib.id and
+              lib.id = :library_id
+         order by title, author
+        """
```

Comment:
Minor: There is a lot of duplication between these queries, it would be nice to not have to duplicate so much code here.